src/js/view/components/LogViewer.js

```diff
@@ -135,24 +167,23 @@ const LogViewer = props => {
       });
       previousLinesLength.current = props.logs.length;
 
-      // Checking if the follow switch is on and if the log file is running.
+      // Checking if the follow switch is on and if the log file is running, then keep the scrollbar at the base to follow.
       if (tailSwitch && logFileHasRunningStatus) {
-        setScrollPosition(0);
+        setScrollPosition(minScrollPositionValue);
       }
     }
   }, [props.logs]);
 
   useEffect(() => {
-    // Effect for checking if tailswitch is on. If it is - lock the sliderPosition to 0 and display the tail of the file.
-    const APPROXIMATE_AMOUNT_OF_BYTES_TO_FETCH = 10000;
-    const LINES_FROM_VIEWCONTAINER_BOTTOM = 5;
-
+    // Effect for checking if tailswitch is on. If it is, and the file is running - lock the sliderPosition to the base and display the tail of the file.
+    const BYTE_AMOUNT_TO_FETCH =
+      (props.nrOfLinesInViewer - 3) * meanByteValueOfCurrentLines;
```

Comment:
Maybe this `3` can be replaced with a constant so it is easier to glance it's meaning?